Makefile

```diff
@@ -4,8 +4,10 @@ ROOTDIR=$(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 # Base path used to install.
 DESTDIR=/usr/local
 
-# Used to populate version variable in main package.
-VERSION=$(shell git describe --match 'v[0-9]*' --dirty='.m' --always)
```

Comment:
Please don't remove this. We are going to use tags to set the version.

Do you want us to setup a 1.0.0-dev tag? I was going to do this when submitting the versioning plan.